src/shared/dropdown-list-v2/dropdown-list.tsx

```diff
@@ -359,6 +375,7 @@ export const DropdownList = <T, V>({
                 <ListItem
                     aria-selected={selected}
                     aria-multiselectable={multiSelect}
+                    aria-disabled={hasSelectedMax as boolean}
```

Comment:
instead of doing type casting here, you can do `hasSelectedMax = !!maxSelectable && !!selectedItems ...` to cast the variables to boolean